syntax = "proto3";

import "trade/vo/Order.proto";
option java_package = "com.huasheng.sa.quant.open.sdk.protobuf.trade.response";
option java_outer_classname = "QueryTodayOrderListResponseProto";
message QueryTodayOrderListResponse {
    repeated Order orderList = 1;   //当日订单列表
    string pageQueryKey = 2; //分页查询参数
    bool hasNext=3; //是否还有下一页
}